tests/web3js/config.js

```diff
@@ -1,6 +1,6 @@
 const { Web3 } = require('web3')
 
-const web3 = new Web3("http://localhost:8545")
+const web3 = new Web3("http://127.0.0.1:8545")
```

Comment:
This change is needed because the GH test runner probably uses IPv4, instead of IPv6.